### DISCUSSION TITLE
Latex formula to wide for the latex field

### DISCUSSION THREAD

**Discussion by LovisR** [xournalpp/xournalpp#4935] [Q&A]
So I have a formula that I want to put into my document, it's this:
`P(g|w,a,p)=\frac{P(w)P(a|p,w)P(p)P(s|w,a,p)P(g|s)}{P(w)P(a|p,w)P(p)\left(P(s|w,a,p)P(g|s)+P(\lnot s|w,a,p)P(g|\lnot s)+P(s|w,a,p)P(\lnot g|s)+P(\lnot s|w,a,p)P(\lnot g|\lnot s)\right)}`

But this cuts off at the halfway point when I try to generate it with the Add/Edit Tex tool.

Any way I can make the field the tool creates wider?

![Screenshot from 2023-06-18 14-57-02](https://github.com/xournalpp/xournalpp/assets/26107824/bca37270-e6aa-4065-b1d4-06b0afd5193d)

**Comment by Soothfast** [comment]
So you have the complete code down there in the black field.  When clicking OK what happens?

**Comment by tattsan** [comment]
This will be fixed by #5264. You can modify your latex template file as in #5264.

**Reply by tattsan** [reply]
#5264 has been merged into  Xournal++ 1.2.3. Please try this release.
